src/Uno.UWP/ApplicationModel/Appointments/AppointmentManager.Android.cs

```diff
@@ -0,0 +1,62 @@
+﻿#if __ANDROID__

```

Comment:
```suggestion
#nullable enable

﻿#if __ANDROID__

```

Reply:
It cannot be nullable, as RequestStoreAsync returns AppointmentStore, not AppointmentStore?, and can return null.

Reply:
Nullables don't change the API definition, so you should be able to make the change.